pkg/services/devicegroups.go

```
@@ -154,6 +158,40 @@ func (s *DeviceGroupsService) GetDeviceGroupByID(ID string) (*models.DeviceGroup
 	return &deviceGroup, nil
 }
 
+// GetDeviceGroupDetailsByID gets the device group details by ID from the database
+func (s *DeviceGroupsService) GetDeviceGroupDetailsByID(ID string) (*models.DeviceGroupDetails, error) {
+	var deviceGroupDetails models.DeviceGroupDetails
+	account, err := common.GetAccountFromContext(s.ctx)
+	if err != nil {
+		s.log.WithField("error", err.Error()).Error("Error account")
```

@acosferreira Is this the correct way to handle the error?

---

You should return the error as well to stop the process, we can't continue without an account.
i.e return nil, err